CHANGELOG.md

```diff
@@ -59,6 +59,8 @@ release.
   ([#252](https://github.com/open-telemetry/semantic-conventions/pull/252))
 - Simplify HTTP metric briefs.
   ([#276](https://github.com/open-telemetry/semantic-conventions/pull/276))
+- Move non-`network.*` attributes out of network.yaml.
+  ([#296](https://github.com/open-telemetry/semantic-conventions/pull/296))
```

Comment:
@joaopgrassi Why did you request this changelog entry? This should be 100% transparent to users and and just an internal detail. If we want want to have every PR in the changelog we should just auto-generate it. I would remove this again.

Reply:
It was more of a question, not a request per se. I thought about the case where people might look for for them in the yaml and not finding and maybe then looking for the reason of why. Having it in the changelog could help discovering things.

Reply:
I personally didn't think it was necessary, but IMO it's up to @open-telemetry/specs-semconv-maintainers to decide where they want to draw the line when it comes to adding change log entries.

Reply:
Sorry for the confusion here folks, I should have indicated in my comment why I thought a changelog would come in handy. I think it makes sense for the case I stated (mostly discovering things) but also OK with me without it.